texts/formatted_enceladus.txt

```diff
@@ -0,0 +1,1611 @@
+;	"find" vblex pprs
+	"find" vblex subs
+;	"find" vblex ger
+"<these>"
+	"this" det dem pl
+;	"this" prn dem mf pl
+"<bacteria>"
+	"bacterium" n pl
+"<a lot>"
+	"a lot" adv
+"<easier>"
+	"easy" adj sint comp
+"<,>"
+	"," cm
+"<without>"
+;	"without" adv
+	"without" pr
+"<needing>"
+;	"need" vblex pprs
+	"need" vblex subs
+;	"need" vblex ger
+"<to>"
+	"to" pr
+"<drill>"
+;	"drill" n sg
+	"drill" vblex inf
+;	"drill" vblex pres
+;	"drill" vblex imp
+"<through>"
+	"through" pr
+"<kilometres>"
+	"kilometre" n pl
+"<of>"
+	"of" pr
+"<ice>"
+	"ice" n sg
+"<and>"
+	"and" cnjcoo
+"<rocks>"
+	"rock" n pl
+;	"rock" vblex pres p3 sg
+"<.>"
+	"." sent
+"<Cassini>"
+	"Cassini" np cog sg
+"<wasn’t>"
+;	"not" adv
+;		"be" vbser past p1 sg
+	"not" adv
+		"be" vbser past p3 sg
+"<ready>"
+	"ready" adv
+;	"ready" adj sint
+"<to>"
+	"to" pr
+"<try>"
+	"try" vblex inf
+;	"try" vblex pres
+;	"try" vblex imp
+"<to>"
+	"to" pr
+"<collect>"
+	"collect" vblex inf
+;	"collect" vblex pres
+;	"collect" vblex imp
+"<these>"
+	"this" det dem pl
+;	"this" prn dem mf pl
+"<possible>"
+	"possible" adj
+"<alien>"
+	"alien" n sg
+"<life>"
+	"life" n sg
+"<forms>"
+	"form" n pl
+;	"form" vblex pres p3 sg
+"<,>"
+	"," cm
+"<but>"
+	"but" cnjcoo
+;	"but" pr
+"<the>"
+	"the" det def sp
+"<analysis>"
+	"analysis" n sg
+"<of>"
+	"of" pr
+"<Enceladus>"
+	"Enceladus" np top sg
+"<’>"
+;	"'" apos
+	"'s" gen
+"<plumes>"
+	"plume" n pl
+"<revealed>"
+;	"reveal" vblex pp
+	"reveal" vblex past
+"<various>"
+	"various" adj
+;	"various" det ind pl
+"<molecules>"
+	"molecule" n pl
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<are>"
+	"be" vbser pres
+"<associated with>"
+;	"associate# with" vblex past
+	"associate# with" vblex pp
+"<life>"
+	"life" n sg
+"<:>"
+	":" sent
+"<water>"
+	"water" n sg
+;	"water" vblex inf
+;	"water" vblex pres
+;	"water" vblex imp
+"<vapour>"
+	"vapour" n sg
+"<,>"
+	"," cm
+"<carbon dioxide>"
+	"carbon dioxide" n sg
+"<,>"
+	"," cm
+"<methane>"
+	"methane" n sg
+"<,>"
+	"," cm
+"<nitrogen>"
+	"nitrogen" n sg
+"<,>"
+	"," cm
+"<propane>"
+	"propane" n sg
+"<,>"
+	"," cm
+"<acetylene>"
+	"acetylene" n sg
+"<,>"
+	"," cm
+"<formaldehyde>"
+	"formaldehyde" n sg
+"<and>"
+	"and" cnjcoo
+"<traces>"
+	"trace" n pl
+;	"trace" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<ammonia>"
+	"ammonia" n sg
+
+"<.>"
+	"." sent
+
+
+"<In>"
+;	"in" adv
+	"in" pr
+"<2015>"
+	"2015" num
+"<,>"
+	"," cm
+"<Cassini>"
+	"Cassini" np cog sg
+"<performed>"
+;	"perform" vblex pp
+	"perform" vblex past
+"<a>"
+	"a" det ind sg
+"<close>"
+;	"close" n sg
+	"close" adj sint
+;	"close" vblex inf
+;	"close" vblex pres
+;	"close" vblex imp
+"<flyby>"
+	"flyby" n sg
+"<of>"
+	"of" pr
+"<Enceladus>"
+	"Enceladus" np top sg
+"<’>"
+;	"'" apos
+	"'s" gen
+"<south>"
+;	"south" adv
+;	"south" n sg
+	"south" adj sint
+"<pole>"
+	"pole" n sg
+"<,>"
+	"," cm
+"<passing>"
+;	"passing" n sg
+;	"pass" vblex pprs
+;	"pass" vblex subs
+	"pass" vblex ger
```

Comment:
I think this should be a present participle?

Reply:
According to the [guidelines](http://wiki.apertium.org/wiki/Tagging_guidelines_for_English#Verbs_with_.22-ing.22), it's present participle if it functions like an adjective. Not the case here